src/coreclr/pal/inc/pal.h

```diff
@@ -2707,8 +2663,9 @@ PALIMPORT BOOL PALAPI PAL_GetUnwindInfoSize(SIZE_T baseAddress, ULONG64 ehFrameH
 #define PAL_CS_NATIVE_DATA_SIZE 48
 #elif defined(__linux__) && defined(__loongarch64)
 #define PAL_CS_NATIVE_DATA_SIZE 96
+#elif defined(__linux__) && defined(_riscv) && __riscv_xlen == 64
+#define PAL_CS_NATIVE_DATA_SIZE 96
```

Comment:
@janvorli, I was testing this on different platforms and noticed that `sizeof(PAL_CRITICAL_SECTION_NATIVE_DATA)` on linux-arm64 is 104, instead of 116.

Reply:
Thanks for checking it, we can update it then.